src/NewFamilyMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import './NewFamilyMember.scss';
+
+enum DocumentType {
+  type,
+  dni,
+  nie,
+  pasaporte
+};
+
+function NewFamilyMember() {
+  const [name, setName] = useState<string>('');
+  const [surname, setSurname] = useState<string>('');
+  const [date, setDate] = useState<string>('');
+  const [documentType, setDocumentType] = useState<DocumentType>(DocumentType.type);
```

Comment:
```suggestion
  const [documentType, setDocumentType] = useState<DocumentType>();
```

```suggestion
  const [documentType, setDocumentType] = useState<DocumentType | null>(null);
```

If the first suggestion doesn't work, I would recommend adding null as a possible type instead of having an empty default value